go/oasis-node/cmd/debug/fixgenesis/fixgenesis.go

```diff
@@ -252,6 +348,39 @@ func computeStakeClaims(
 	return computedStakeClaims, nil
 }
 
+func updateBeaconGenesis(old *v4Document) (beacon.Genesis, error) {
+	oldP := &old.Beacon.Parameters // Save some typing.
+
+	// Check to see if we know how to set this up in a sensible manner.
+	if oldP.DebugMockBackend || oldP.InsecureParameters != nil || oldP.DebugDeterministic {
+		return beacon.Genesis{}, fmt.Errorf("existing document has debug/insecure beacon parameters set")
+	}
+	if oldP.Backend != "pvss" {
+		return beacon.Genesis{}, fmt.Errorf("existing document is not using the PVSS beacon")
+	}
+
+	// Derive "reasonable" parameters based off the existing parameters.
+	pvssP := oldP.PVSSParameters
+	alphaHighQualityThreshold := uint64(pvssP.Participants)
```

Comment:
Is this sufficient or should it be increased?

Reply:
It's a tinfoil hattery parameter.  In theory, I am being over-cautious because there is no way for a single node to derive the other node's beta values (ie: the best that a single node can do even if granted control over alpha is to try to game it so that their beta gives them a statistical advantage in the sortition).

Reply:
Ok, makes sense, thanks!